app/src/main/java/com/spoony/spoony/core/designsystem/component/topappbar/SpoonyBasicTopAppBar.kt

```diff
@@ -16,40 +18,46 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.spoony.spoony.R
 import com.spoony.spoony.core.designsystem.theme.SpoonyAndroidTheme
+import com.spoony.spoony.core.util.extension.noRippleClickable
 
-@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun SpoonyBasicTopAppBar(
     modifier: Modifier = Modifier,
     showBackButton: Boolean = false,
     backgroundColor: Color = SpoonyAndroidTheme.colors.white,
     onBackButtonClick: () -> Unit = {},
-    actions: @Composable RowScope.() -> Unit = {},
+    actions: @Composable () -> Unit = {},
     content: @Composable () -> Unit = {}
 ) {
-    CenterAlignedTopAppBar(
-        title = content,
-        modifier = modifier,
-        navigationIcon = {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween,
+        modifier = modifier
+            .background(backgroundColor)
+            .fillMaxWidth()
+            .padding(vertical = 12.dp)
+            .statusBarsPadding()
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier
+                .weight(1f)
+        ) {
             if (showBackButton) {
-                IconButton(
-                    onClick = onBackButtonClick
-                ) {
-                    Icon(
-                        imageVector = ImageVector.vectorResource(R.drawable.ic_arrow_left_24),
-                        contentDescription = null,
-                        tint = SpoonyAndroidTheme.colors.gray700,
-                        modifier = Modifier
-                            .size(32.dp)
-                    )
-                }
+                Icon(
+                    imageVector = ImageVector.vectorResource(R.drawable.ic_arrow_left_24),
+                    contentDescription = null,
+                    tint = SpoonyAndroidTheme.colors.gray700,
+                    modifier = Modifier
+                        .size(32.dp)
+                        .noRippleClickable(onBackButtonClick)
+                        .padding(start = 12.dp)
+                )
```

Comment:
이 부분 피그마 확인한 바로는 아이콘 크기는 24/24이지만, 터치영역을 32/32로 원하는 것 같던데 맞을까요?
![image](https://github.com/user-attachments/assets/96301ac7-3596-49ed-8dc2-97ed2eca834f)
지금 코드상의 터치영역은 이렇게 되어있는 것 같습니다.
그리고 아이콘 크기도 작아져 있는 것 같아요.


```suggestion
 Box(
                    contentAlignment = Alignment.Center,
                    modifier = Modifier
                        .padding(start = 12.dp)
                        .size(32.dp)
                        .clickable(onClick = onBackButtonClick)
                ) {
                    Icon(
                        imageVector = ImageVector.vectorResource(R.drawable.ic_arrow_left_24),
                        contentDescription = null,
                        tint = SpoonyAndroidTheme.colors.gray700,
                        modifier = Modifier
                    )
                }
```
![image](https://github.com/user-attachments/assets/b679c96e-5b4f-47d1-a519-d606363bfa5e)

이렇게 한번 래핑해서 사용하는건 어떨까요? 사실 `IconButton` 컴포넌트가 저런 구조를 사용하고 있습니다ㅎㅎ

Reply:
와 터치영역 놓칠뻔했네요 꼼꼼한 감사합니다😊😊 말씀하신대로 아이콘 크기가 24.dp, 터치 영역은 32.dp가 맞아요..!! 그리고 확인해보니 아이콘 크기는 피그마에 있던 그대로 24.dp로 적용되어 있습니다!! 아마 프리뷰라서 작아보였던 것이 아닐까 싶네요??

IconButton이 Icon을 Box로 감싼 구조인거 처음 알았어요ㅎㅎ 사실 개인적으로는 사용하지 않아도 구현이 가능한 컴포저블 함수들은 최대한 쓰지 않는 것을 좋아하는데 이 경우 제안해주신대로 Box로 감싸는 것이 가독성 측면에서 이점이 커서 이렇게 수정하도록 하겠습니다!!

Reply:
아 그리고 IconButton 내부 코드 보다가 발견했는데요 Material3 문서를 보시면 아이콘 버튼의 터치 영역은 최소 48.dp여야 해서 IconButton도 최소크기가 48.dp로 되어있더라구요ㅎㅎ

알게 된 점 1: 우리 서비스는 Material Design을 따르지 않고 있어요
알게 된 점 2: 그래서 IconButton을 쓰면 디자인과 맞지 않을 것 같네요ㅎㅎ

여러분 모두 이거 참고하기!!!

Reply:
맞아요 `IconButton` 은 내부 패딩을 고정시키고 선택 범위도 지들 맘대로 해놔서 저도 잘 안쓰긴합니다ㅋㅋ
구현 방식을 참고하면 좋을 것 같아서 말씀드렸습니다ㅎㅎ 이것도 아티클 보다가 내부구현 보게된건데 링크를 못찾아서ㅠ